app/src/main/kotlin/com/wire/android/ui/common/CurrentSessionViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.wire.android.ui.common
+
+import androidx.lifecycle.ViewModel
+import com.wire.android.util.ui.WireSessionImageLoader
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class CurrentSessionViewModel @Inject constructor(
+    val wireSessionImageLoader: WireSessionImageLoader
+): ViewModel()
```

Comment:
It's a dummy ViewModel so we can inject the `ImageLoader` into composables.

Reply:
Right now, even if we use `WireActivityViewModel`, it will instantiate a `WireActivityViewModel` for each composable using it. So it's... not much of an improvement.

I'll see about creating a custom Dagger component/scope next week.

This way we will have a solid scope for each user session, and we can have a singleton image loader inside of it.